javascript/chronometer.js

```diff
@@ -1,34 +1,48 @@
 class Chronometer {
-  constructor() {
-    // ... your code goes here
-  }
-
+  constructor() {}
+  currentTime = 0;
+  intervalId = null;
   start(callback) {
```

Comment:
I see that just now. Interesting that it works. Normally it has to be like this:
```
 constructor() {
    this.currentTime = 0;
    this.intervalId = null;
  }
```